spec/models/service_template_transformation_plan_task_spec.rb

```diff
@@ -329,11 +329,14 @@
 
     context 'source is vmwarews' do
       let(:src_ems) { FactoryBot.create(:ems_vmware, :zone => FactoryBot.create(:zone)) }
-      let(:src_host) { FactoryBot.create(:host_vmware_esx, :ext_management_system => src_ems, :ipaddress => '10.0.0.1') }
-      let(:src_storage) { FactoryBot.create(:storage, :ext_management_system => src_ems, :name => 'stockage récent') }
 
-      let(:src_lan_1) { FactoryBot.create(:lan) }
-      let(:src_lan_2) { FactoryBot.create(:lan) }
+      let(:vmware_cluster) { FactoryBot.create(:ems_cluster, :ext_management_system => src_ems) }
+      let(:src_host) { FactoryBot.create(:host_vmware_esx, :ems_cluster => vmware_cluster, :ext_management_system => src_ems, :ipaddress => '10.0.0.1') }
+      let(:src_storage) { FactoryBot.create(:storage, :hosts => [src_host], :name => 'stockage récent') }
```

Comment:
Please don't specify `:name`, it's sequenced by the factory.

Reply:
if this `:name` is for `:name => 'stockage récent'`, it is needed for verification by subsequent specs,